examples/web-socket-events/src/main/java/com/iland/app/App.java

```diff
@@ -125,10 +126,11 @@ static void webSocketProcessEvents() {
     final SocketSubscription orgSubscription =
         webSocketClient.consumeEvents(orgEventConsumer, orgEventFilter);
 
-    // Keeping the process alive so the web socket can consume events.
-    // Note that there is no error handling for interrupted or connection errors
-    // and that you should add try/catch block to handle such cases.
-    while (true) {
+    // Keeping the process alive for 5 minutes so the web socket can consume events.
+    try {
+      Thread.sleep(TimeUnit.MINUTES.toMillis(5));
+    } catch (final InterruptedException e) {
+      log.error(e.getMessage());
```

Comment:
you shouldn't catch the InterruptedException here, let it cause the program to exit